README.md

```diff
@@ -20,8 +20,9 @@ Files will be downloaded to this path on the remote server before being moved in
 
     solr_create_user: true
     solr_user: solr
+    solr_group: solr
```

Comment:
Can you update this default to be `solr_group: "{{ solr_user }}"` so it's more obvious that that's the default?

Reply:
Sure!

Reply:
I followed the same model from the README.md ```solr_install_path: /opt/solr```
which is ```solr_install_path: "/opt/{{ solr_service_name }}"``` in defaults/mail.yml